src/Libraries/CoreNodes/String.cs

```diff
@@ -38,6 +38,25 @@ public static object ToNumber(string @string)
             throw new ArgumentException(Properties.Resources.StringToNumberInvalidNumberMessage, "str");
         }
 
+        /// <summary>
+        /// Get all of the number strings from the target string as a string
+        /// </summary>
+        /// <param name="string">Target string to be get</param>
+        /// <returns name="str">Number In string</returns>
+        /// <search>getnumber,tonumber,strtonumber,numberinstring,string2number,stringtonumber,int,double,cast</search> 
+        public static string GetNumber(string @string)
+        {
+            string str = string.Empty;
+            if (string.IsNullOrEmpty(@string)) return str;
+            List<double> result = new List<double>();
+            for (int i = 0; i < @string.Length; i++)
+            {
+                if (char.IsDigit(@string[i]))
+                    str += @string[i];
```

Comment:
this might be a good case to use a string builder - because this string could be arbitrarily long.
https://michaelscodingspot.com/challenging-the-c-stringbuilder/